src/FtpClient.php

```diff
@@ -1062,6 +1062,10 @@ public function copyToLocal(string $remoteSource, string $destinationFolder) : b
 
             $files = $this->listDirDetails($remoteSource, true);
             foreach ($files as $file) {
+                if (substr($file['path'], 0, 2) !== './') {
```

Comment:
The `$file['path']` should have the `./` on the first two characters because it will be matched on following regular expression with `preg_match`.

Otherwise it will not be matched.

Reply:
That okay.